Documentation/swagger-extensions.md

```diff
@@ -15,6 +15,7 @@ The following documents describes AutoRest specific vendor extensions for [Swagg
 * [x-ms-discriminator-value](#x-ms-discriminator-value) - maps discriminator value on the wire with the definition name.
 * [x-ms-client-flatten](#x-ms-client-flatten) - flattens client model property or parameter.
 * [x-ms-parameterized-host](#x-ms-parameterized-host) - replaces the Swagger host with a host template that can be replaced with variable parameters.
+* [x-ms-mutability](#x-ms-mutability) - provides insight to Autorest on how to generate code. It doesn't alter the modeling of what is actually sent on the wire.

```

Comment:
@amarzavery : Can we add this documentation in rest-api-specs repo? We decided to add swagger related documentation in that repository right?